quick-start/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	vault "github.com/hashicorp/vault/api"
+)
+
+const (
+	vaultServerAddr = "http://127.0.0.1:8200"
+	token           = "dev-only-token" // This is the root token for our Vault dev server. Don't do this in production!
+	pathToSecret    = "secret/data/my-secret-password"
+	secretKey       = "password"
+	secretValue     = "Hashi123"
+)
+
+func main() {
+	config := vault.DefaultConfig()
+
+	config.Address = vaultServerAddr
+
+	client, err := vault.NewClient(config)
+	if err != nil {
+		log.Fatalf("unable to initialize Vault client: %v", err)
+	}
+
+	// Authentication
+	client.SetToken(token)
+
+	secretData := map[string]interface{}{
+		"data": map[string]interface{}{
+			secretKey: secretValue,
+		},
+	}
+
+	// Writing a secret
```

Comment:
```suggestion
	// Write a secret
```

(I think present tense is more common in comments)